warehouse/query-core/src/main/java/datawave/query/jexl/visitors/RegexIndexExpansionVisitor.java

```diff
@@ -266,14 +263,7 @@ public Object visit(ASTERNode node, Object data) {
                     log.info(logLine);
                 }
                 
-                // If we've *never* seen this field, we want to denote the difference against it not being indexed
-                if (fieldName.equals(Constants.ANY_FIELD)) {
-                    return node;
-                } else if (!allFields.contains(fieldName)) {
-                    return RebuildingVisitor.copyInto(node, ASTUnknownFieldERNode.create());
```

Comment:
I am a little scared we are dropping something.  Do we have a replacement for the ASTUnknownFieldERNode and ASTUnsatisfiableERNode ?

Reply:
In the old code we create ASTUnsatisfiableERNode instances, but nowhere in the code will you find an example where we check to see if a node is an ASTUnsatisfiableERNode.  So, it was effectively unused.  

The existence of ASTUnknownFieldERNode triggered special logic in one place - RangeStream.  If you check RangeStream, you will see that I moved the check being performed in the RegexIndexExpansionVisitor to that visitor in order to trigger the creation of an unknown field scanner stream.  This was the only place where this type was checked for, so this should be equivalent.    

Also, keep in mind that these custom datawave JexlNode types do NOT survive if you print the jexl tree as a string, and parse that string back into a tree again - so this kind of logic is fairly brittle.  Jexl doesn't know what these nodes are, so to me it made sense to move away from custom types and stick with the primitives.